src/desktop/remote_desktop.rs

```diff
@@ -138,6 +148,25 @@ pub enum Axis {
     Horizontal = 1,
 }
 
+#[cfg_attr(feature = "glib", derive(glib::Enum))]
+#[cfg_attr(feature = "glib", enum_type(name = "AshpdRemoteDesktopPersistMode"))]
+#[derive(Default, Serialize_repr, PartialEq, Eq, Debug, Copy, Clone, Type)]
+#[doc(alias = "XdpRemoteDesktopPersistMode")]
+#[repr(u32)]
+/// Persistence mode for a remote desktop session.
+pub enum PersistMode {
+    #[doc(alias = "XDP_REMOTEDESKTOP_PERSIST_MODE_NONE")]
+    #[default]
+    /// Do not persist.
+    DoNot = 0,
+    #[doc(alias = "XDP_REMOTEDESKTOP_PERSIST_MODE_TRANSIENT")]
+    /// Persist while the application is running.
+    Application = 1,
+    #[doc(alias = "XDP_REMOTEDESKTOP_PERSIST_MODE_PERSISTENT")]
+    /// Persist until explicitly revoked.
+    ExplicitlyRevoked = 2,
+}
```

Comment:
Not a fan of this duplication. Just copy the enum to src/desktop/mod.rs and re-export it from screencast (with a deprecated warning) to avoid breaking existing code

Reply:
Okay, I moved PersistMode into src/desktop/mod.rs. Unfortunately, it seems you can't add deprecated warnings to re-exports https://github.com/rust-lang/rust/issues/30827.

Reply:
Oh really, that is unfortunate.

Reply:
Then, just remove the re-export as we can't land this in 0.8.x anyways.